apps/tutorial-servo-control-can/TutorialServo.cpp

```diff
@@ -0,0 +1,48 @@
+//TutorialServo Source file
+
+#include "mbed.h"
+#include "TutorialServo.h"
+
+class TutorialServo
```

Comment:
Class definition, `public` and `private` declarations should only be in header file

See _MarsRover2020-firmware/lib/controllers/Motor.h_ and _MarsRover2020-firmware/lib/controllers/Motor.cpp_ for reference.

Reply:
Right, how did I miss that, will update